src/webcomponents/study/admin/study-create.js

```diff
@@ -118,22 +118,22 @@ export default class StudyCreate extends LitElement {
         this.#setLoading(true);
         this.opencgaSession.opencgaClient.studies()
             .create(this.study, {project: this.project.fqn})
-            .then(res => {
+            .then(() => {
                 this.study = {};
                 this._config = this.getDefaultConfig();
                 NotificationUtils.dispatch(this, NotificationUtils.NOTIFY_SUCCESS, {
                     title: "Study Create",
                     message: "New study created correctly"
                 });
-                LitUtils.dispatchCustomEvent(this, "sessionUpdateRequest");
+                LitUtils.dispatchCustomEvent(this, "sessionUpdateRequest", {}, {}, error);
+                LitUtils.dispatchCustomEvent(this, "studyCreate", {}, {}, error);
```

Comment:
At this point, `error` is not defined yet (because it is assigned in the `catch`), why do you need to pass it to both events dispatchers?

Reply:
My bad on the cut&paste...